project/plugins.sbt

```diff
@@ -10,3 +10,5 @@ addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.5.2")
 addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.6.4")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-test-report" % "1.5.0")
+
+addSbtPlugin("org.playframework" % "sbt-plugin" % "3.0.5")
```

Comment:
we don't want to pull in play if we can help it 🤞

Reply:
Done